pkg/operation/botanist/component/istio/istiod.go

```diff
@@ -176,10 +176,7 @@ func (i *istiod) Destroy(ctx context.Context) error {
 var TimeoutWaitForManagedResource = 2 * time.Minute
 
 func (i *istiod) Wait(ctx context.Context) error {
-	timeoutCtx, cancel := context.WithTimeout(ctx, TimeoutWaitForManagedResource)
```

Comment:
> Wait can't be used

why not?

Reply:
The MR never gets healthy as it needs loadbalancer IP for one of it's service.

Reply:
I cannot follow, can you share some related code, please?

Reply:
If I use wait, istio mr fails to get ready with the following error 
```
'Service "istio-ingress/istio-ingressgateway" is unhealthy: service is
      missing ingress status'
```
The status is missing because it can't get any loadbalancer IP assigned to it. And the reason for it is, there is no ingress controller running which is part of extension controller only in case of provider local.

Reply:
I still don't understand this - the `.status.loadBalancer.ingress` field is not maintained by `Ingress` controllers but by the `Service` controllers part of `cloud-controller-managers.